test/integration/global_web_socket_monitor/vm_test.dart

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@TestOn('browser')
+@TestOn('vm')
```

Comment:
wait why was the vm_test set to test on browser before?

Reply:
It was probably just a copy-paste mistake, but it didn’t matter because we were using those test-runner files that pulled in all of the individual test files and they had the correct `@TestOn` annotations. Now that I’ve removed those in favor of running the individual test files directly, this became a problem and had to be fixed